lib/ClangImporter/ClangModuleDependencyScanner.cpp

```diff
@@ -265,17 +265,34 @@ ModuleDependencyVector ClangImporter::bridgeClangModuleDependencies(
       swiftArgs.push_back(IncludeTree);
     }
 
+    // Ensure we get the "real", vfs-remapped path to the modulemap
+    // becuase it is used as an explicit input for the resulting compilation
+    // task.
+    auto externalModuleMapPathRef =
+        getClangInstance().getFileManager().getFileRef(
+            clangModuleDep.ClangModuleMapFile);
+    StringRef externalModuleMapPath;
+    if (externalModuleMapPathRef.takeError()) {
```

Comment:
This will assert, because the error is not handled; you need to use `llvm::consumeError` if you want to ignore an `Error`.  But, if you don't care about the error value/message you could switch to `getOptionalFileRef` to just get an `optional<FileEntryRef>` instead.

Reply:
I added handling/consuming for this error with `toString` to append the message to the warning diagnostic that got added here. Thanks.